common/requirements.txt

```diff
@@ -15,8 +15,7 @@ django==2.0.2
 docutils==0.14            # via botocore
 idna==2.6                 # via requests
 jmespath==0.9.3           # via boto3, botocore
-pip==9.0.3
```

Comment:
Why did you remove pinning pip? Are we sure we're entirely pip10 compatible?

Reply:
I can't think of any way we wouldn't be, all the images build successfully and the tests pass.

I figure we've gotta remove it at some point, we can't stay on that version forever.